### PR TITLE
NCL-1479 Product Import Process: move creating of product entity to t…

### PR DESCRIPTION
**Pull request by alexcreasy**
…he UI

### CODE REVIEW

**Comment by jbartece** [comment]
Looks good to me, but I'm not UI pro

**Comment by alexcreasy** [comment]
I think we're all repurposed Java geeks here ;)

**Comment by vibe13** [comment]
@alexcreasy @jbartece  looks good to me also

**Comment by thescouser89** [comment]
Hi, it fails for me because of this:

https://github.com/project-ncl/pnc/blob/1b8db11939316744d0ddc848a529e2a8b07ed463/ui/app/import/product/ProductImportCtrl.js#L283

I guess 'name' should be 'id'

File: ui/app/import/product/ProductImportCtrl.js [hunk 1]
```diff
@@ -282,7 +282,7 @@
           scope.startSubmitDisabled = true;
           if (_(r).has('name')) {
```
Comment (thescouser89) [inline ui/app/import/product/ProductImportCtrl.js]: This 'name' should be 'id'.

Comment (alexcreasy) [inline ui/app/import/product/ProductImportCtrl.js]: @thescouser89 good spot!

**Comment by alexcreasy** [comment]
Retest this please

**Comment by thescouser89** [comment]
https://github.com/project-ncl/pnc/blob/master/ui/app/import/product/ProductImportCtrl.js#L307

Has to be changed too! Sorry for not finding this out earlier!

**Comment by alexcreasy** [comment]
Retest this please

**Comment by thescouser89** [comment]
:+1:

**Comment by alexcreasy** [comment]
merging!

**Pull request merged**